pkg/ipam/ipam.go

```diff
@@ -91,8 +91,12 @@ type K8sEventRegister interface {
 	K8sEventProcessed(scope string, action string, status bool)
 }
 
+type MtuConfiguration interface {
+	GetDeviceMTU() int
```

Comment:
Why not make this method part of the `pkg/ipam/ipam.go:Configuration` interface and avoid having to pass down a new argument to all functions?

Reply:
I'm not sure I understand your proposal. `pkg/ipam/ipam.go:Configuration` is an interface implemented by `option.Config`, which does not contain the necessary information to obtain the MTU.

Reply:
Oh that is unfortunate, I thought `mtuConfig` was part of DaemonConfig :-(